src/libraries/NonZeroDeltaCount.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.20;
+
+import {IHooks} from "../interfaces/IHooks.sol";
```

Comment:
was there a reason to split it into two files now? just bc nonzerodelta count is not per locker anymore?

Reply:
oops just seen this sorry - yeah just because it feels like very separate things they feel unrelated